language/golang/language.go

```diff
@@ -112,10 +112,11 @@ func (l *Language) ExecuteTests(logger *log.Logger, repositoryPath string) (test
 
 		StdOut: commandOutput,
 	}
-	testResult.Coverage, err = language.CoverageObjectCountOfFile(logger, coverageFilePath)
+	coverageData, err := language.ParseCoverage(logger, coverageFilePath)
 	if err != nil {
 		return testResult, problems, pkgerrors.WithMessage(pkgerrors.WithStack(err), commandOutput)
 	}
+	testResult.Coverage = uint64(language.CountUniqueCoverage(coverageData))
```

Comment:
why not put that into a helper? we now have the same code three times.

Reply:
It's just a single line ._. also I think it gave me conflicts rearranging the commits.

Reply:
It's 5 lines, isn't it? from `language.ParseCoverage` to `uint64(language.CountUniqueCoverage(coverageData))`

If rearranging the commits is too much effort, just do it as a last commit.